en/airframes/adding_a_new_frame.md

```diff
@@ -1,25 +1,26 @@
-# Adding a new Airframe Configuration
+# Adding a New Airframe Configuration
 
 PX4 uses canned configurations as starting point for airframes. Adding a configuration is straightforward: Create a new file which is prepended with a free autostart ID in the [init.d folder](https://github.com/PX4/Firmware/tree/master/ROMFS/px4fmu_common/init.d) and [build and upload](../setup/building_px4.md) the software.
 
 Developers not wanting to create their own configuration can instead customize existing configurations using text files on the microSD card, as detailed on the [custom system startup](../advanced/system_startup.md) page.
 
-## Airframe configurations
+## Airframe Configurations
 
 An airframe configuration consists of three main blocks:
 
   * The apps it should start, e.g. multicopter or fixed wing controllers
```

Comment:
@bkueng This is just the bit ` set MAV_TYPE 1` right? Any other "apps" are automatically set from whatever number you enter? 
- Is there a list of numbers
- should we call this "The controllers that should be started, e.g.. ..." instead?
- Perhaps move this below to match order of declaration in config file?

Also, should we add "documentation" in this list - it is important!

Reply:
For this example it's just `MAV_TYPE`, yes.
- List of numbers: it's the `MAV_TYPE` param and @dagar added the full param description for the numbers in a PR: https://github.com/PX4/Firmware/pull/8001/files#diff-86d388773b907f1fd874d2990d901c8dR79 (It matches the definition in mavlink: http://mavlink.org/messages/common).
- It's more than controllers. For example it's also the land detector that depends on the type. And the Intel Aero starts a specific ESC driver: https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/4070_aerofc#L71
- Makes sense